src/SFML/Window/JoystickManager.cpp

```diff
@@ -112,7 +112,7 @@ JoystickManager::JoystickManager()
 ////////////////////////////////////////////////////////////
 JoystickManager::~JoystickManager()
 {
-    for (Item& item : m_joysticks)
+    for (const Item& item : m_joysticks)
```

Comment:
@eXpl0it3r How can I supress the linter here? This can be const on Android, but not on any other platform.

Reply:
Yeah we can do that. Fix the rest of the CI pipeline then I can help with clang-tidy in this one particular circumstance.

Reply:
It's ready.

Reply:
@ChrisThrasher Hi, how do you prefer to resolve that lint issue? `// NOLINT`, `ifdef` for Android, or some other way?

Reply:
Hi @ChrisThrasher @eXpl0it3r , sorry to bother you, I'd like to fix that linter error so the PR is ready for review. As mentioned previously, linter wants me to make the Item ref const, since all relevant methods in Android joystick impl are const. But that only applies to the Android platform as the others are implemented in different ways.

Reply:
```cpp
// NOLINTNEXTLINE(the clang check you want to silence goes here)
```